lib/index.js

```diff
@@ -216,6 +220,13 @@ exports.init = function (sequelize, optionsArg) {
 
   function createBeforeHook(operation) {
     var beforeHook = function beforeHook(instance, opt) {
+      if (opt.noPaperTrail) {
```

Comment:
Where is this flag coming from?

Reply:
It can be set on the save method of the instance to not log the current action

Reply:
So this is if you want to skip the revision updates for a particular action? Let's add this to the README as well 👍

Reply:
Yes, for example when there is no user logged in, but the userid column is required in the schema

Reply:
I'll do the changes tomorrow